tests/adapters/switches/dell_test.py

```diff
@@ -1532,6 +1532,44 @@ def test_unset_bond_mtu(self):
 
         self.switch.unset_bond_mtu(10)
 
+    def test_get_mac_addresses(self):
+        flexmock(self.switch.page_reader).should_receive("do") \
+            .with_args(self.mocked_ssh_client, "show bridge address-table").once() \
+            .and_return(
+            [
+                "",
+                "Aging time is 300 Sec"
+                "",
+                "  Vlan        Mac Address       Port          Type",
+                "-------- --------------------- ---------- --------------------",
+                "1201     F8B1.5649.0F99        vlan 1201  Management",
+                "1206     F8B1.5649.0F99        vlan 1206  Management",
+                "3991     02E0.5236.3801        1/g22      Dynamic",
+                "3991     748E.F8A7.1B01        ch1        Dynamic",
+                "4063     0013.C601.53A7        2/g22      Dynamic",
+                "4063     00C0.B785.3213        ch1        Dynamic",
```

Comment:
why do we discard the mac of the port channel ?
shouldn't it be a conscious decision eg. type!= management, rather than the regex breaking on those ports.
IMO, the regex should parse each line, but we discriminate them either against the port or the management type.

Reply:
definitely en error. will fix.